test/e2e/build_test.go

```diff
@@ -59,6 +59,18 @@ var _ = Describe("Podman build", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	// following test must trigger for both remote and non-remote
+	It("podman build with a secret from file", func() {
+		session := podmanTest.Podman([]string{"build", "-f", "build/Dockerfile.with-secret", "-t", "secret-test", "--secret", "id=mysecret,src=build/secret.txt", "build/"})
```

Comment:
Can we add (another) test with more than 1 secret?

Reply:
added in latest commit.